### PR TITLE
Kubediff deployment 

### PR DESCRIPTION
**Pull request by samiyaakhtar**
Including kubediff deployment as part of azure-simple

### CODE REVIEW

File: cluster/common/flux/deploy_flux.sh [hunk 1]
```diff
@@ -58,6 +58,9 @@ fi
 # back to the roor dir
 cd ../../../../
 
+echo "KUBECTL CONFIG GET-CONTEXTS"
```
Comment (andrebriggs) [inline cluster/common/flux/deploy_flux.sh]: @samiyaakhtar I understand why we added this for debugging purposes in the deploy_kubediff.sh file but we probably don't need this here.

File: cluster/environments/azure-simple/variables.tf [hunk 2]
```diff
@@ -15,6 +15,11 @@ variable "agent_vm_count" {
     default = "3"
 }
 
+variable "agent_vm_size" {
```
Comment (andrebriggs) [inline cluster/environments/azure-simple/variables.tf]: @samiyaakhtar is this a merge from master?

File: cluster/common/kubediff/deploy_kubediff.sh [hunk 3]
```diff
@@ -0,0 +1,60 @@
+#!/bin/sh
```
Comment (yradsmikham) [inline cluster/common/kubediff/deploy_kubediff.sh]: maybe add a rm-rf /kubediff to remove kubediff content before shell scripts run

File: cluster/environments/azure-simple/variables.tf (hunk 2 above)
Comment (samiyaakhtar) [reply cluster/environments/azure-simple/variables.tf]: Yes it was due to a merge, I will have to undo it

[the remark this answers is not in the document]
Comment (samiyaakhtar) [reply cluster/environments/azure-simple/variables.tf]: It was due to a merge from master, will remove

File: cluster/common/kubediff/deploy_kubediff.sh (hunk 3 above)
Comment (samiyaakhtar) [reply cluster/common/kubediff/deploy_kubediff.sh]: Done

**Pull request merged**